code/game/objects/items/weaponry.dm

```diff
@@ -224,7 +224,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	resistance_flags = FIRE_PROOF
 
 /obj/item/katana/cursed
-	slot_flags = null
+	slot_flags = null | NODROP

```

Comment:
NODROP is a part of item_flags, not slot_flags

Reply:
My shift clicking is off...